source/game.game.bmx

```diff
@@ -775,7 +775,7 @@ Type TGame Extends TGameBase {_exposeToLua="selected"}
 
 		Local boss:TPlayerBoss = GetPlayerBoss(playerID)
 		boss.Initialize()
-		boss.creditMaximum = difficulty.creditMaximum
+		boss.creditMaximum = difficulty.creditMinimum
```

Comment:
Ist es wirklich ein "Minimum"? Der Boss kann ja auch weniger anbieten. Ich haette es als "Maximum zu diesem Zeitpunkt" angesehen ...

Reply:
Eigentlich ist es weder ein Minimum noch ein Maximum, da der Boss je nach Laune noch einen Faktor verwendet.
Der Konfigurationswert ist insofern ein Minimum, da dieser Grundwert immer als Kredit zur Verfügung steht und der firmenwertspezifische Anteil noch draufaddiert wird.
Man könnte es höchstens noch in "initialCredit" umbenennen.

Reply:
Anstatt zu addieren könnte man natürlich auch max(initialCredit, nettoWorthBonus) verwenden.

Reply:
Ich würde es gern bei creditMinimum belassen, da es der globale minimale Kreditrahmen ist.